tasks.py

```diff
@@ -216,8 +220,22 @@ def notify(ctx):
         username=env.get_credential('FEC_SLACK_BOT', 'fec-bot'),
     )
 
-def run_migrations(ctx, space):
-    print("\nMigrating database...")
-    jdbc_url = to_jdbc_url(os.getenv('FEC_MIGRATOR_SQLA_CONN_{0}'.format(space.upper())))
+@task
+def create_sample_db(ctx):
+    """
+    Load schema and data into the empty database pointed to by $SQLA_SAMPLE_DB_CONN
+    """
+
+    print("Loading schema...")
+    db_conn = os.getenv('SQLA_SAMPLE_DB_CONN')
+    jdbc_url = to_jdbc_url(db_conn)
+    run_migrations(ctx, jdbc_url)
+    print("Schema loaded")
+    print("Loading sample data...")
+    subprocess.check_call(
+        ['psql', '-v', 'ON_ERROR_STOP=1', '-f', 'data/sample_db.sql', db_conn],
+    )
+    print("Sample data loaded")
+
+def run_migrations(ctx, jdbc_url):
```

Comment:
Looks like we're missing the `@task` decorator here if the intention is to be able to run this directly from the shell as an `invoke` task.

Reply:
I was not intending for this to be run independently as an `invoke` task, but now that you mention it, it may be nice to have.